pipeline/README.md

```diff
@@ -4,20 +4,20 @@
 
 ## Bootstrapping
 
-Initial install for a cluster must happen in fybrik-system (currently).  It won't hurt anything if you aren't sure, and reinstall in fybrik-system.
-```
-. source-external.sh
-bash -x bootstrap-pipeline.sh fybrik-system
-# follow on screen instructions
-```
+Run the following command to bootstrap a Fybrik instance. By default the instance of Fybrik will be deployed as namespaced scoped, meaning the controller will only watch and process FybrikApplications from a specific namespace. Therefore limiting you to create FybrikApplications from one namespace. By that namespace is the one specific on the bootstrap script (a.k.a *system* namespace). You can change this behavior by setting environment variables. For details see [Scoping the deployment](#scoping-the-deployment).
```

Comment:
@roee88 Do we want this to be the default behavior for open source?

Reply:
This is the default for the pipeline installation. I think it's fine.

Reply:
By default pipeline will setup namespace scoped app controller.